tools/generator-go-sdk/generator/templater_methods_autorest.go

```diff
@@ -869,7 +870,7 @@ func (c methodsAutoRestTemplater) optionsStruct(data ServiceGeneratorData) (*str
 	headerAssignments := make([]string, 0)
 
 	for optionName, option := range c.operation.Options {
-		optionType, err := golangTypeNameForObjectDefinition(option.ObjectDefinition)
+		optionType, err := golangTypeNameForObjectDefinition(option.ObjectDefinition, &data.commonPackageName)
```

Comment:
options will always be local to the package, they should never be common types since it's specific to an operation (e.g. header/querystring)?